packages/neovim/src/api/Buffer.test.ts

```diff
@@ -216,7 +220,7 @@ describe('Buffer API', () => {
         expect(await buffer.getOption('copyindent')).toBe(false);
 
         // Restore option
-        buffer.setOption('copyindent', initial);
+        buffer.setOption('copyindent', initial!);
```

Comment:
nit: could at assert(initial) on line 217, that might be a better pattern in general.

Reply:
Use assert for type narrowing? I didn't know that. Thanks for teaching me something new!